recipe/meta.yaml

```
@@ -12,8 +12,8 @@ source:
     - 0001-use-conda-ar-not-system.patch
 
 build:
-  number: 0
-  skip: true  # [(not linux) or cuda_compiler_version in (undefined, "None", "12.6")]
+  number: 2
+  skip: true  # [ppc64le or (not linux) or cuda_compiler_version in (undefined, "None")]
```

Could we please leave in `ppc64le` given it is [still supported upstream]( https://github.com/NVIDIA/nccl/blob/be8bf056e81786b5e147bcdee54cbdb9a76a01be/src/graph/xml.cc#L495-L496 )?

```suggestion
  skip: true  # [(not linux) or cuda_compiler_version in (undefined, "None")]
```